src/get-started/codelab.md

```diff
@@ -345,6 +353,12 @@ Process finished with exit code 0
       "UpperCamelCase".
     {{site.alert.end}}
 
+    {{site.alert.important}}
+    Using pascal case also helps screen readers identify the individual words 
+    in the compound word and provides a better 
+    experience to visually impaired users.
+    {{site.alert.end}}
```

Comment:
**issue:** Refer to this capitalization format as **Upper Camel Case** per the [Dart style guide](https://dart.dev/guides/language/effective-dart/style#do-name-types-using-uppercamelcase). Remove the mention of "pascal case".